firebase_admin/_user_mgt.py

```diff
@@ -331,6 +379,63 @@ def iterate_all(self):
         return _UserIterator(self)
 
 
+class DeleteUsersResult:
+    """Represents the result of the ``auth.delete_users()`` API."""
+
+    def __init__(self, result, total):
+        """Constructs a DeleteUsersResult.
+
+        Args:
+          result: BatchDeleteAccountsResponse: The proto response, wrapped in a
+              BatchDeleteAccountsResponse instance.
```

Comment:
Suggest backticks for literal.

Is the result after the colon above code-fonted as well?

Reply:
> backticks for literal

Done.

> after the colon

Hm. I'm not sure that should even be there. I've removed it and rephrased it slightly.